database/postgres/seed.js

```diff
@@ -0,0 +1,73 @@
+const faker = require('faker');
+const fs = require('fs');
+var i = 1;
+// const projects = [];
+//project row
+const generateOneProjectCSVRow = (id) => {
+  return `${id},` + faker.date.past().toISOString()
+}
+
+const generateOneUpdateCSVRow = (id, foreignLimit) => {
+  return `${id},"${faker.random.words()}",${JSON.stringify(faker.lorem.paragraphs())},${faker.date.past().toISOString()},${faker.random.number()},${Math.ceil(Math.random()*foreignLimit)}`
+}
+
+const generateOneCommentCSVRow = (id, foreignLimit) =>{
+  return `${id}, ${faker.random.number()},${faker.image.avatar()},${faker.internet.userName()},${faker.date.recent().toISOString()},"${faker.lorem.sentence()}",${Math.ceil(Math.random()*foreignLimit)}`
+}
+
+
+// const writeOneTable = function writeOneTable(dataNumber, stream, dataGen) {
+//   let loop = dataNumber;
+//   const writer = function writer() {
+//     let ok = true;
+//     do {
+//       loop--;
+//       if (loop === 0) {
+//         stream.write(JSON.stringify(dataGen(loop)), 'utf8');
+//       } else {
+//         ok = stream.write(JSON.stringify(dataGen(loop)), 'utf8');
+//       }
+//     } while (loop > 0 && ok);
+//     if (loop > 0) {
+//       stream.once('drain', writer);
+//     }
+//   };
+//   writer();
+// };
+
```

Comment:
Let's get rid of unnecessary comments here :)

Reply:
done